uiplib/settings.py

```diff
@@ -6,6 +6,16 @@
 HOME_DIR = os.path.join(os.path.expanduser("~"), '.uip')
 NUMBER_OF_IMAGES_TO_PARSE = 15
 DEFAULT_PICS_FOLDER = os.path.join(HOME_DIR, 'pics')
+DEFAULT_SETTINGS = {'timeout': 30*60,
```

Comment:
hey we decalare this dict again on `setup.py` can you make the changes so we dont write it there and just use it there as well?